### PR TITLE
News: Add Dec DJ, 1 sw release, 1 coverage item

### PR DESCRIPTION
**Pull request by xaur**
**Rebased on #1106 ** which made structural changes to the .yml files.

Not submitting any Jan coverage because I didn't map it out yet. We have a slow process but I can't make it faster atm.

### CODE REVIEW

**Comment by xaur** [comment]
Guessed again on the icon use. Can correct if needed.

File: src/data/news/software_releases.yml [hunk 1]
```diff
@@ -1,288 +1,222 @@
+- 
+  Title: "Bison Relay Release v0.1.3"
+  Permalink: "https://github.com/companyzero/bisonrelay/releases/tag/v0.1.3"
+  Params:
+    sortDate: "2023-01-20 21:40:16""
```
Comment (jholdstock) [inline src/data/news/software_releases.yml:5]:
```suggestion
    sortDate: "2023-01-20 21:40:16"
```

This is causing the build to fail

**Review by jholdstock — CHANGES_REQUESTED** [review]
Looks fine apart from the quotation mark typo

File: src/data/news/software_releases.yml (hunk 1 above)
Comment (xaur) [reply src/data/news/software_releases.yml]: Oops fixed. Good to have the build is checking this.

**Review by dajohi — APPROVED** [review]

**Pull request merged**